bindings/jupyroot/CMakeLists.txt

```diff
@@ -75,6 +75,12 @@ foreach(val RANGE ${how_many_pythons})
                              LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries
                              ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT libraries)
 
+  if (NOT MSVC AND NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR)
```

Comment:
Hi @chrisburr thanks for taking care of this!

The solution of the symlink works, but looking at the other libraries (`libROOTPythonizations`, `libcppyy` and `libcppyy_backend`), it's only done when the library is used both from Python and C++, which is the case only for `libcppyy` and `libcppyy_backend`, because cppyy uses them but also TPython.

Bottom line: I believe this should be done like for `libROOTPythonizations`:

https://github.com/root-project/root/blob/master/bindings/pyroot/pythonizations/CMakeLists.txt#L152-L153

where there is no symlink but instead, when installing the library, we use `${CMAKE_INSTALL_PYTHONDIR}` instead of `${CMAKE_INSTALL_LIBDIR}`.